packages/main/test/specs/SegmentedButton.spec.js

```diff
@@ -77,6 +77,19 @@ describe("SegmentedButton general interaction", () => {
 		assert.ok(await segmentedButtonItem2.matches(":focus"), "The selected SegmentedButtonItem should be focused.");
 	});
 
+	it("tests disabled focus", async () => {
+		const button1 =  await browser.$("#button1");
+		const button2 =  await browser.$("#button2");
+		const segmentedButtonItem1 =  await browser.$("#testSB1ToggleBtn");
+
+		await button1.click();
+		await button1.keys("Tab");
+		assert.ok(await segmentedButtonItem1.matches(":focus"), "The first SegmentedButtonItem should be focused.");
+
+		await button2.keys("ArrowRight");
```

Comment:
I don't get this test. I think button2 is a ui5-button, and we execute ArrowRight on a ui5-button. Is it expected to move the focus in any case?
Perhaps executing ArrowRight on the focused segmented button item makes more sense as it will test that the focus will not move to the disabled one.

Reply:
It's missed with copying the logic, but it worked. I guess executing it on the button, actually executes it as a regular arrow right. It fails without the fix and works with the fix, maybe that's why it was missed. But anyway, I will fix the element we execute on.